src/configManager/models/configHandler.js

```diff
@@ -0,0 +1,88 @@
+const dbConnector = require('./database/databaseConnector');
+const constConfig = require('../../common/consts').CONFIG;
+const logger = require('../../common/logger');
+
+const configDataMap = {
+    [constConfig.GRFANA_URL]: { value: process.env.GRAFANA_URL },
+    [constConfig.EXTERNAL_ADDRESS]: { value: process.env.EXTERNAL_ADDRESS, default: process.env.INTERNAL_ADDRESS },
+    [constConfig.INTERNAL_ADDRESS]: { value: process.env.INTERNAL_ADDRESS },
+    [constConfig.DOCKER_NAME]: { value: process.env.DOCKER_NAME, default: 'zooz/predator-runner:latest' },
+    [constConfig.JOB_PLATFORM]: { value: process.env.JOB_PLATFORM },
+    [constConfig.RUNNER_CPU]: { value: process.env.RUNNER_CPU, default: 1, type: 'int' },
+    [constConfig.RUNNER_MEMORY]: { value: process.env.RUNNER_MEMORY, default: 2048, type: 'int' },
+    [constConfig.METRICS_PLUGIN_NAME]: { value: process.env.METRICS_PLUGIN_NAME, type: 'json' },
+    [constConfig.METRICS_EXPORT_CONF]: { value: process.env.METRICS_EXPORT_CONFIG, type: 'json' },
+    [constConfig.SMTP_SERVER]: {
+        value: JSON.stringify({
+            host: process.env.SMTP_HOST,
+            port: process.env.SMTP_PORT,
+            username: process.env.SMTP_USERNAME,
+            password: process.env.SMTP_PASSWORD,
+            timeout: process.env.SMTP_TIMEOUT
+        }),
+        type: 'json'
+    }
+};
+
```

Comment:
1. extract all configuration to one file. (recommendation)
2. I dont see any advantage to save it like key - object. why not simple like key value? 
for example:    
    [constConfig.DOCKER_NAME]: process.env.DOCKER_NAME || 'zooz/predator-runner:latest'